build-support/bin/generate_github_workflows.py

```diff
@@ -64,52 +64,45 @@ class Platform(Enum):
 DISABLE_REMOTE_CACHE_ENV = {"PANTS_REMOTE_CACHE_READ": "false", "PANTS_REMOTE_CACHE_WRITE": "false"}
 
 
-IS_PANTS_OWNER = "${{ github.repository_owner == 'pantsbuild' }}"
+IS_PANTS_OWNER = "github.repository_owner == 'pantsbuild'"
 
 # ----------------------------------------------------------------------
 # Actions
 # ----------------------------------------------------------------------
 
+_DOCS_ONLY_TEXT = "DOCS_ONLY"
+
+
+def _docs_only_cond(docs_only: bool) -> str:
+    op = "==" if docs_only else "!="
+    return f"needs.docs_only_check.outputs.docs_only {op} '{_DOCS_ONLY_TEXT}'"
+
 
 def is_docs_only() -> Jobs:
     """Check if this change only involves docs."""
     linux_x86_64_helper = Helper(Platform.LINUX_X86_64)
+    docs_files = ["docs/**"]
```

Comment:
Would it make sense to circle back now that the action can handle something like `**/*.md` to add this generic pattern to catch / encourage markdown in the source tree along with markdown edits?

Reply:
Yeah, I think that could make a lot of sense! Probably after we move off readme.com, since their syncing logic expects a certain directory structure. But if we switch to something where we get to control the layout mapping, we can probably have the best of both worlds.

Reply:
I was actually thinking its good and fine to have docs that are not even part of a doc site. Generally notes to people who actually hack on Pants itself.

Reply:
That does make sense!